### PR TITLE
Fyle doc fixes to change v1beta to v1

### PR DESCRIPTION
**Pull request by sumanth-fyle**

### CODE REVIEW

File: docs/concepts/errors.md [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+*   Codes in the 5xx range indicate an error with Fyle's servers(these are rare). 
 
-This section is yet to be written
+> Please contact us at support@fylehq.com if a 5xx error is encountered. 
+
+## Types Of Errors
+| SL NO.  |  HTTP STATUS CODE  | HTTP MESSAGE | ENGLISH EXPALINATION |
+|---------|--------------------|--------------|--------------------- |
+|    1.   |         400       |   Bad Request |  The request data or query sent is malformed |
+|     2.   |         401      |  Unauthorized |  The token is either not given Fyle or is timed out |
```
Comment (snarayanank2) [inline docs/concepts/errors.md:14]: Explanation is wrong

File: docs/concepts/errors.md [hunk 2]
```diff
@@ -1,3 +1,60 @@
...
+    |:--:|
+    | <b>Example of ValidationError. Required parameter user_full_name is missing</b>    
+2. **BulkError:**
+    This type error occurs when you are doing a bulk operations. Generally this error occurs when you pass a value for an entity parameter for which that particular entity is not found. (Eg: Passing an unkonwn department name which does not exists etc.). 
+     <!--focus: false-->
+    |![Required Parameter Missig Error](../../assets/images/concepts/errors/bulkError.png)|
+    |:--:|
+    | <b>Example of BulkError. Department named XYZ does not exists in the org</b> 
+3. **IntegrityError:**
+    This error is encountered when an sql contraints fails. (Eg: Creating an project with same project name , sub project name inside the same org, passing user_id for which the users dont exists).
```
Comment (snarayanank2) [inline docs/concepts/errors.md:33]: Don't mention this as SQL constraint.. we should be cleaning up these and not passing them back as is. External developer can't access our DB

File: docs/concepts/errors.md [hunk 3]
```diff
@@ -1,3 +1,60 @@
...
+    <!--focus: false-->
+    |![Required Parameter Missig Error](../../assets/images/concepts/errors/integrityError.png)|
+    |:--:|
+    | <b>Example of IntegrityError. User is trying to create a project with same values. Hene unqiue constraint failed</b>
```
Comment (snarayanank2) [inline docs/concepts/errors.md:37]: Hene unqiue

**Review by snarayanank2 — CHANGES_REQUESTED** [review]

File: docs/concepts/errors.md (hunk 2 above)
Comment (sumanth-fyle) [reply docs/concepts/errors.md:33]: I will just remove explanation for these kinds of error

**Review by snarayanank2 — APPROVED** [review]

**Pull request merged**